test_xdb/models/under_test/generate_uuid_test.sql

```diff
@@ -1,4 +1,12 @@
-{{ config({"tags":["exclude_bigquery", "exclude_bigquery_tests"]}) }}
+{{ config({
+    "tags":["exclude_bigquery", "exclude_bigquery_tests"],
+    "pre-hook": [{"sql": '{%- if target.type == "postgres" -%}
+                          CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
```

Comment:
What's the reason for adding this extra step in here? Was the test not passing beforehand?

Reply:
I think, it’s not really necessary, it matters only if you are testing this way:
1-st terminal: `docker-compose up `
2-nd terminal: 
`docker-compose exec testxdb bash`
`dbt run --profiles-dir . --target postgres`
In this case you will get an error related to Postgres extension.
But if you run this command only (as pipeline does):
`docker-compose exec testxdb test postgres`
It will be OK because of `prepare_postgres_db()` function.